src/module/entities/helper.js

```diff
@@ -246,7 +246,7 @@ module.exports = class UserProjectsHelper {
 					status: CONSTANTS.common.ACTIVE_STATUS,
 				}
 				// Specify the fields to include in the result set
-				const userRoleExtensionProjection = ['_id', 'title', 'userRoleId', 'userType']
+				const userRoleExtensionProjection = ['_id', 'title', 'userRoleId']
```

Comment:
@MallanagoudaB userType also we can pass. I think it won't break anything in front end.